src/Eshopworld.Tests.Core/Eshopworld.Tests.Core.csproj

```diff
@@ -4,9 +4,9 @@
     <TargetFramework>netstandard2.1</TargetFramework>
     <IsPackable>true</IsPackable>
 
-    <Version>2.0.5</Version>
-    <PackageReleaseNotes>Updated xUnit trait attribute to be applicable at class level.</PackageReleaseNotes>
-
+    <Version>2.0.7</Version>
+    <PackageReleaseNotes>Update Eshopworld.Tests.Core.Analyzers.</PackageReleaseNotes>
+    <PackageIcon>icon.png</PackageIcon>
```

Comment:
Reason for double bump is that 2.0.5 was brought in from recovering the old commit.
Current version of nuget is 2.0.6 hence bump to 2.0.7